packages/app/pages/AllProjectsPage/AllProjectsPage.tsx

```diff
@@ -6,10 +6,12 @@
  * @since:   2017-08-07
  */
 
-import * as React from 'react'
-import * as themeable from 'react-themeable'
-import BindAgent, { AllOrgsType } from '@vflows/bindings/agent/allOrganizations'
-import Link from '@vflows/views/atoms/Link'
+import * as React from "react"
```

Comment:
Why did you change the syntax here?

Reply:
I didn't. It just changed to double quotes to align with the linter